### PR TITLE
refactor: simplified start proj logs

### PR DESCRIPTION
**Pull request by iamsahilsonawane**
## Change
Improved the _how to navigate and start your project_ logs after initialization.

## Why this change?

In the previous state, it was harder for users:
1. To know that these are commands
2. To copy and paste that commands into the terminal

## Inspiration 

While creating Google Flutter's projects, it has a similar structure which makes it easier for copy-pasting in the terminal. When I came to fresh for the first time, it was a little hard for me to quickly catch them as commands.

## Comparison (Before | After)
image

### CODE REVIEW

**Comment by xyntechx** [comment]
Great idea! I saw that you're still using `unresolvedDirectory` for the `cd` command, which is good! The right-hand-side image you sent show `cd hello_world` though... it should be `cd my-project` if it's using `unresolvedDirectory` 😁

Minor nitpick but I hope it's just the image that's slightly wonky!

**Comment by iamsahilsonawane** [comment]
> Great idea! I saw that you're still using `unresolvedDirectory` for the `cd` command, which is good! The right-hand-side image you sent show `cd hello_world` though... it should be `cd my-project` if it's using `unresolvedDirectory` 😁
> 
> Minor nitpick but I hope it's just the image that's slightly wonky!

Actually no, the unresolved directory gives us the name of the directory in which the project is been initialized. I created a project named, `hello_world`.

If you see the new updated code (not yet released), when you create a new project, you get the name of the directory in which you created the project. 

So it's nothing changed, it's still the unresolved directory shown there in the 2nd image. Basically instead of giving us resolved directory which will be for e.g. `Users/<username>/.../directory` it gives us an unresolved directory which will be `directory`. `my-project` is a static log previously which soon got updated in this https://github.com/denoland/fresh/pull/415 pull.

**Comment by xyntechx** [comment]
> > Great idea! I saw that you're still using `unresolvedDirectory` for the `cd` command, which is good! The right-hand-side image you sent show `cd hello_world` though... it should be `cd my-project` if it's using `unresolvedDirectory` 😁
> > Minor nitpick but I hope it's just the image that's slightly wonky!
> 
> Actually no, the unresolved directory gives us the name of the directory in which the project is been initialized. I created a project named, `hello_world`.
> 
> If you see the new updated code (not yet released), when you create a new project, you get the name of the directory in which you created the project.
> 
> So it's nothing changed, it's still the unresolved directory shown there in the 2nd image. Basically instead of giving us resolved directory which will be for e.g. `Users/<username>/.../directory` it gives us an unresolved directory which will be `directory`. `my-project` is a static log previously which soon got updated in this #415 pull.

Ah I see! Thanks for the clarification!

File: init.ts [hunk 1]
```diff
@@ -386,5 +386,6 @@ await generate(resolvedDirectory, manifest);
 // Specifically print unresolvedDirectory, rather than resolvedDirectory in order to
 // not leak personal info (e.g. `/Users/MyName`)
 console.log("\n%cProject created!", "color: green; font-weight: bold");
-console.log(`\`cd ${unresolvedDirectory}\` to enter to the project directory.`);
-console.log("Run \`deno task start\` to start the development server.");
+console.log(`\nIn order to start the development server, type:\n`);
```
Comment (lucacasonato) [inline init.ts]:
```suggestion
console.log(`\nIn order to start the development server, run:\n`);
```

**Review by lucacasonato — APPROVED** [review]
Thank you.

**Pull request merged**